src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -47,7 +47,7 @@
         "symfony/form": "~2.8|~3.0",
         "symfony/expression-language": "~2.8|~3.0",
         "symfony/process": "~2.8|~3.0",
-        "symfony/serializer": "~2.8|^3.0",
+        "symfony/serializer": "~3.2",
```

Comment:
We tried to preserve compatibility with old versions of the Serializer component for a while. I'm not against this change (it will allow to simplify the code), but a lot of class_exists checks can then be removed (can be done in another PR though).